maven-plugins/helidon-archetype-maven-plugin/pom.xml

```diff
@@ -115,6 +115,9 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-invoker-plugin</artifactId>
+                <configuration>
+                    <streamLogs>true</streamLogs>
```

Comment:
The maven-invoker-plugin is used in various modules and the configuration is quite complicated. The configuration is centralized in the top-level pom to minimize the project level configuration as well as to guaranty the same behavior.

What was your reasoning for this change ?

Reply:
It was important during the tests. I removed it.